codecov/settings_base.py

```diff
@@ -572,3 +572,11 @@
     "reports": "shared.django_apps.reports.migrations",
     "legacy_migrations": "shared.django_apps.legacy_migrations.migrations",
 }
+
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
+        "LOCATION": get_config("services", "redis_url"),
+        "TIMEOUT": 300,  # 5 min TTL
+    }
+}
```

Comment:
I originally chose django's local memory cache, until I read this part of the docs:

> Note that each process will have its own private cache instance, which means no cross-process caching is possible. This also means the local memory cache isn’t particularly memory-efficient, so it’s probably not a good choice for production environments. It’s nice for development.

Sidenote: sometime we import `cached_property` from `functools` and sometimes from `django.utils.functional`. It seems like the django one [is better](https://stackoverflow.com/questions/68593165/what-is-the-difference-between-cached-property-in-django-vs-pythons-functools)? 

I'll just use redis directly instead of this django cache abstraction then